src/kits/tracker/ContainerWindow.cpp

```diff
@@ -562,6 +526,28 @@ BContainerWindow::BContainerWindow(LockingList<BWindow>* list,
 		list->AddItem(this);
 	}
 
+	fRootLayout = new BGroupLayout(B_VERTICAL, 0);
+	fRootLayout->SetInsets(0);
+	SetLayout(fRootLayout);
+	fRootLayout->Owner()->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
+
+	if (!fIsDeskWindow) {
```

Comment:
Actually, this happens because we don't want to add the titleview, countview, or scrollbars when we are a desktop window. The same logic exists in the pre-layout code, just not in the constructor, as there were no layouts to create.